tests/sqlparser_common.rs

```diff
@@ -15015,3 +15015,8 @@ fn parse_set_time_zone_alias() {
         _ => unreachable!(),
     }
 }
+
+#[test]
+fn parse_return() {
+    all_dialects().verified_stmt("RETURN");
```

Comment:
Can we have this check assert the AST? Also we can add another test case that includes a return statement having an expression

Reply:
> Can we have this check assert the AST?

Done 👍 

> Also we can add another test case that includes a return statement having an expression

Done, although this is invalid for SQL Server. I think having that test here would also mean we can't have a test that asserts an error when return with value is found as a top level statement. Thoughts? 🤔

Reply:
hmm yeah given that its currently part of the parser behavior I think its reasonable to assert it. If the parser later stops accepting a top-level return value then we would change the test here to assert an error instead